### PR TITLE
Change consent policy state enum

### PR DESCRIPTION
**Pull request by zhouyx**
Learned my lesson:  not to use `0` for enum value. Try to fix this before it's too late : )

### CODE REVIEW

File: src/consent-state.js [hunk 1]
```diff
@@ -21,10 +21,10 @@ import {Services} from './services';
...
  */
 export const CONSENT_POLICY_STATE = {
-  UNKNOWN: 0,
-  SUFFICIENT: 1,
-  INSUFFICIENT: 2,
-  UNKNOWN_NOT_REQUIRED: 3,
+  UNKNOWN: 1,
+  SUFFICIENT: 2,
+  INSUFFICIENT: 3,
+  UNKNOWN_NOT_REQUIRED: 4,
```
Comment (lannka) [inline src/consent-state.js]: please document this enum will have external dependency, we should never change values anymore (after this PR).

Comment (zhouyx) [reply src/consent-state.js]: done.

**Comment by zhouyx** [comment]
the `amp-ad-0.1.js` size didn't increase after the change. we are good with importing `CONSENT_POLICY_STATE`

**Comment by zhouyx** [comment]
Not sure about why type check fails.
is there any recent change to type check? @rsimha could you help me with this.

**Comment by rsimha** [comment]
@zhouyx No recent changes to type check, so far as I know.
1. Is the path accurate?
2. Are the types being exported properly from the source file?
3. Are the type names being imported properly in the file that's reporting the Type check errors?

**Comment by zhouyx** [comment]
Thanks @rsimha 
I think I know why check type is failing. `3p/adsense.js` imports from `consent-state.js`, which imports `Services` from `'./services.js`. 
I should separate the enum value to a different file to fix this.

**Comment by zhouyx** [comment]
@lannka The change is indeed risky. Almost forget the ima-video 😨

**Comment by zhouyx** [comment]
@choumx @jridgewell could you help with the v0.js size? I don't think the change will add size to it.

**Comment by jridgewell** [comment]
> Try to fix this before it's too late

What’s the bug? There’s nothing inherently wrong with using zero, as long as all checks are `===`.

> could you help with the v0.js size

Just bump the size.

**Comment by zhouyx** [comment]
It's not a bug now, but error prone. As you mention as long as all checks are `===`. Since this an enum exposing to 3P code, we probably want to avoid that. 

Cool! I'll bump the size!

**Comment by rsimha** [comment]
Percy diff send to be from a new video test that's failing. @agassemi?

**Comment by aghassemi** [comment]
@rsimha Looks like the video "controls" get hidden based on a timer and depending on when Percy takes a screenshot, it can flake. https://github.com/ampproject/amphtml/pull/15479 to fix it. For now we can approve the existing screenshots to make this PR green.

**Pull request merged**